model/src/main/kotlin/OrtResult.kt

```diff
@@ -55,6 +55,15 @@ data class OrtResult(
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
         val data: CustomData = emptyMap()
 ) {
+    /**
+     * Conveniently return the declared licenses for the given [id] which may either refer to a project or to a package.
+     */
+    fun getDeclaredLicensesForId(id: Identifier) =
```

Comment:
Same applies as for `getDetectedLicensesForPackage` in the previous commit, why not return `null` if the `id` is not found?
Also, the naming is inconsistent, use either `...ForId` or `...ForPackage` (I'd prefer the first).

Reply:
I see, any reason not to also check the projects in `getDetectedLicensesForPackage`?

Reply:
Before we scan, [projects are implicitly converted to packages](https://github.com/heremaps/oss-review-toolkit/blob/master/downloader/src/main/kotlin/Downloader.kt#L70). So when asking for detected licenses there are only packages, no more projects. But since we *do* effectively consider projects here, how about renaming this to `ForId`, too?

Reply:
Yes, that sounds good.